ui/send_page.go

```diff
@@ -912,9 +946,21 @@ func (pg *SendPage) watchForBroadcastResult() {
 	}
 }
 
-func (pg *SendPage) changeEvt(evt widget.EditorEvent) {
+// handleEditorChange tracks changes on the editor and adjust its width of the send amount input field
+// it also updates the DCR - USD exchange rate value
+func (pg *SendPage) handleEditorChange(evt widget.EditorEvent) {
```

Comment:
Why does a single method handle events for both the amount editor and the destination address editor?

Reply:
How do you mean address editor? It handles only events in the amount editor.

Reply:
handleEditorChange method  is called on lines 299 and 308 to handle change events on the amount editor and destination address editor. On change event of the address field, it calls calculateNextWidth function and calls the GetUSDExchangeValues as well. Shouldn't these methods on be called on change event of the amount field?